### ISSUE TITLE
Unable to move column after hide

### ISSUE THREAD

**Issue by Beercow** [ragardner/tksheet#75]
With the recent update, once you hide a column, you cannot move columns after that. Even if you unhide the columns, moving columns is still broke.

**Comment by ragardner** [comment]
Do you mean using drag and drop with the mouse?

**Comment by Beercow** [comment]
Yes. Shift + left mouse. It works fine before but once you hide a column, it stops working. Even if you unhide them, it’s still broke

**Comment by ragardner** [comment]
Thank you very much for the quick report,

I suspect that the issue is that there has never existed any code to allow drag and drop columns when hiding columns and when unhiding columns using `display_columns()` your `enable` argument was always `True` which meant internally the program thought there were hidden columns, preventing drag and drop after the first use of the function

To remedy this in `5.0.27` I have made it so if `display_columns()` sees that the list of columns is the same as all columns it will internally set `all_columns_displayed` to `True` and drag and drop should work again

I can perhaps make it even more forgiving than that in the next version as well

I will likely work on the code for drag and drop columns while columns are hidden but it may take me a few days or more

tldr: should be fixed in 5.0.27 and will work on allowing drag and drop when columns are hidden but it may take little while

**Comment by Beercow** [comment]
Awesome work. Thank you for your help.

**Comment by ragardner** [comment]
I thought I would let you know, after looking at the code I have decided to complexity is too great to implement column moving while columns are hidden, I think it would require an overhaul of the way tksheet keeps track of cell settings and data in order to make it easy

**Comment by Beercow** [comment]
Thanks for the update. I might tinker with this when I have some time.
